hack/build-go.sh

```diff
@@ -14,14 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script sets up a go workspace locally and builds all go components.
+# This script is a vestigial redirection.  Please do not add "real" logic.
 
 set -o errexit
 set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::golang::build_binaries "$@"
-kube::golang::place_bins
+# For help output
+ARGHELP=""
+if [[ "$#" -gt 0 ]]; then
+    ARGHELP="WHAT='$@'"
+fi
+
+echo "NOTE: $0 has been replaced by 'make' or 'make all'"
+echo
+echo "The equivalent of this invocation is: "
+echo "    make ${ARGHELP}"
+echo
+echo
+make --no-print-directory -C "${KUBE_ROOT}" all WHAT="$*"
```

Comment:
The "default" rule in Make still has to have a name.  It is traditional to call it "all".  in this case I thought it was clearer to be specific.